scripts/resultsSum.pl

```diff
@@ -247,6 +247,12 @@ sub resultReporter {
 		print $fhOut "#SHA.txt content: \n# " . $SHAContent . "\n";
 
 		print $fhOut "# Timestamp: " . $timeStamp . " UTC \n";
+
+		#add customTarget to TAP file
+  		if ($customTarget ne '') {
+  			print $fhOut "# custom target: " . "${customTarget}" . "\n";
```

Comment:
Can we change `custom target` to `CUSTOM_TARGET`, so it matches with the parameter name? Thanks